### ISSUE TITLE
Section on restrictions?

### ISSUE THREAD

**Issue by nschneid** [UniversalConceptualCognitiveAnnotation/docs#31]
There were some restrictions that I learned about today from Omri:

- A scene unit (P/S) cannot be immediately under a D unit.
- A parallel scene (H) cannot have smaller parallel scenes immediately under it.

Maybe these are mentioned somewhere in the guidelines, but I think it would be nice to have a compact description of which categories can occur under which other categories, which categories can be combined on the same unit (like S+A), and so forth.

**Comment by dotdv** [comment]
Yes, definitely, I will try to phrase something.

**Comment by dotdv** [comment]
For now, until we add something to the guidelines here are some basic restrictions: 
D, T, A and G can appear only in Scene units.
E and N can only appear in non-Scene units
Q, R and F can appear both in Scene as well in non-Scene units.
Regarding types of scene units: currently the guidelines allow only for H scenes, A scenes and E scenes.

**Comment by nschneid** [comment]
Great! The thing you mentioned about coordination ("D in coordination", https://github.com/UniversalConceptualCognitiveAnnotation/docs/issues/28#issuecomment-421747396) is an exception, right?

**Comment by dotdv** [comment]
Right.

**Comment by nschneid** [comment]
Is it true that only A, D, and T can be remote? https://github.com/UniversalConceptualCognitiveAnnotation/docs/issues/34#issuecomment-424972891

What about:

- I tried to learn the material 3 times, and so did he.
   I'd guess: [I_A tried_D to_F learn_P [the_E material_C]_A [3_E times_C]_Q]_H and_L [so_F did_F he_A (tried)_D (to)_F (learn)_P [(the)_E (material)_C]_A [(3)_E (times)_C]_Q]_H
   i.e. repeating all the elided material as remote.

**Comment by dotdv** [comment]
> Is it true that only A, D, and T can be remote? #34 (comment)
It's not true, but I didn't mean to say that only As Ds and Ts can be remotes. Sorry if I misled you. I just had these specifically in mind because sometime ago we resolved that Ds and Ts (which were also questioned) should be added just like we add As.  
As for P/S- yes it is definitely added, and if we add Ds and Ts we should also Qs. 
In any case, there is much to clarify as to what should be added and what shouldn't be added in the case of remote.  I think that since the instruction with regards to Remotes was to add only the most minimal unit that is necessary to the conceptualization of the Scene, you probably won't see Fs added. also, it's true that we should add P/S, D, T, A Q  but it doesn't mean that the whole unit is added since it's supposed to be the most minimal unit.

Adding all units as Remotes will probably solve the ambiguity as to what unit is necessary to the conceptualization of the Scene, it's just essentially different from the main instruction so I'll leave it up to Omri to determine.

**Comment by dotdv** [comment]
> Is it true that only A, D, and T can be remote? #34 (comment)

It's not true, but I didn't mean to say that only As Ds and Ts can be remotes. Sorry if I misled you. I just had these specifically in mind because sometime ago we resolved that Ds and Ts (which were also questioned) should be added just like we add As.  
As for P/S- yes it is definitely added, and if we add Ds and Ts we should also Qs. 
In any case, there is much to clarify as to what should be added and what shouldn't be added in the case of remote.  I think that since the instruction with regards to Remotes was to add only the most minimal unit that is necessary to the conceptualization of the Scene, you probably won't see Fs added. also, it's true that we should add P/S, D, T, A Q  but it doesn't mean that the whole unit is added since it's supposed to be the most minimal unit.

> What about:
I tried to learn the material 3 times, and so did he.
I'd guess: [I_A tried_D to_F learn_P [the_E material_C]_A [3_E times_C]_Q]_H and_L [so_F did_F he_A (tried)_D (to)_F (learn)_P [(the)_E (material)_C]_A [(3)_E (times)_C]_Q]_H
i.e. repeating all the elided material as remote.


Adding all units as Remotes will probably solve the ambiguity as to what unit is necessary to the conceptualization of the Scene, it's just essentially different from the main instruction so I'll leave it up to Omri to determine.

**Comment by dotdv** [comment]
> Is it true that only A, D, and T can be remote? #34 (comment)
I didn't mean to say that, sorry if I misled you. 

I wrote you a long reply but deleted it since I just remembered that we changed the Remote section, and I myself am now confused how we should treat Remote in coordination. 

I can just say that your suggestion of repeating all the elided material as remote will probably solve the ambiguity as to what unit is necessary and what unit isn't. But it's an essentially different approach from the principle we followed so far, so I'll leave it up to Omri to determine.

**Comment by dotdv** [comment]
I added a restrictions summary chapter draft to the end of the guidelines. At first I thought it could open chapter 2: "a birds eye's view of the categories" but I wasn't sure about it, so eventually added it to the end by default.

**Comment by nschneid** [comment]
I agree, this is sufficiently technical that it's better to have at the end of the doc.

**Comment by nschneid** [comment]
It says that only H and L can occur at the top-level. What about G, as in: "Unfortunately, to graduate in time, I need to take this course"?

**Comment by omriabnd** [comment]
Sounds good

**Comment by omriabnd** [comment]
By convention we putwithin the scene although it's understood it takes
scope over it.
